### PR TITLE
Refresh against cosmos-sdk's latest master

### PR DESCRIPTION
**Pull request by jgimeno**
______

For contributor use:

- [x] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/gaia/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules/structure.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/gaia/blob/master/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added  relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [x] Re-reviewed `Files changed` in the Github PR explorer

______

For admin use:

- [x] Added appropriate labels to PR (ex. `WIP`, `R4R`, `docs`, etc)
- [x] Reviewers assigned
- [ ] Squashed all commits, uses message "Merge pull request #XYZ: [title]" ([coding standards](https://github.com/tendermint/coding/blob/master/README.md#merging-a-pr))

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -55,6 +55,8 @@ Base64 encoded transactions.
 * [\#164](https://github.com/cosmos/gaia/pull/164) [\#212](https://github.com/cosmos/gaia/pull/212) The LevelDB-based
 keybase implementation has been replaced with a 99 designs Keyring library-backed implementation. Keys created and stored
 with previous `gaia` releases need to be migrated through the `gaiacli keys migrate` command.
+* [\#349](https://github.com/cosmos/gaia/pull/349) The new implementation of the Keyring from SDK does not allow
```
Comment (alessio) [inline CHANGELOG.md]: CHANGELOG might need some extra love. I'll take care of it when this is ready :+1:

Comment (alexanderbez) [reply CHANGELOG.md]: As a long term user/developer of the SDK/Gaia, I don't know what this means.

File: cmd/gaiad/testnet.go [hunk 2]
```diff
@@ -33,6 +24,14 @@ import (
 	"github.com/cosmos/cosmos-sdk/x/genutil"
 	genutiltypes "github.com/cosmos/cosmos-sdk/x/genutil/types"
 	"github.com/cosmos/cosmos-sdk/x/staking"
+	"github.com/spf13/cobra"
```
Comment (alexanderbez) [inline cmd/gaiad/testnet.go]: undo. This is not the correct import structure. Local imports should be grouped by:

- stdlib
- github.com/cosmos/gaia
- everything else

**Review by alexanderbez — CHANGES_REQUESTED** [review]
Thanks @jgimeno! Left some minor feedback 👍

File: cmd/gaiad/testnet.go (hunk 2 above)
Comment (alexanderbez) [reply cmd/gaiad/testnet.go]: If you use VSCode, have the following in your workspace settings:

```json
  "gopls": {
    "local": "github.com/cosmos/gaia",
  },
```

Comment (alessio) [reply cmd/gaiad/testnet.go]: Please use `goimports` @jgimeno

**Comment by jackzampolin** [comment]
Can we maybe rebase this against the `ibc-alpha` branch?

File: CHANGELOG.md [hunk 3]
```diff
@@ -53,7 +53,7 @@ Base64 encoded transactions.
 ### Client Breaking Changes
 
 * [\#164](https://github.com/cosmos/gaia/pull/164) [\#212](https://github.com/cosmos/gaia/pull/212) The LevelDB-based
-keybase implementation has been replaced with a 99 designs Keyring library-backed implementation. Keys created and stored
+keybase implementation has been replaced by a 99designs/keyring library-backed implementation. Keys created and stored
```
Comment (alessio) [inline CHANGELOG.md]:
```suggestion
keybase implementation has been replaced with a 99 designs Keyring library-backed implementation. Keys created and stored
```

File: CHANGELOG.md (hunk 1 above)
Comment (alessio) [reply CHANGELOG.md]: There is no need for a CHANGELOG addition. The migration to the new keyring is already mentioned. Hence resolving this.

File: cmd/gaiad/testnet.go (hunk 2 above)
Comment (alessio) [reply cmd/gaiad/testnet.go]: I've run `make format`. That should do it. Hence resolving.

**Comment by alessio** [comment]
@jackzampolin once this is merged in we could easily cherry-pick the commit

**Review by alexanderbez — APPROVED** [review]
ACK

**Pull request merged**